src/modules/Builder/pages/BuilderApplet/BuilderApplet.schema.ts

```diff
@@ -533,6 +533,9 @@ export const SectionSchema = () =>
     name: yup
       .string()
       .required(getIsRequiredValidateMessage('sectionName'))
+      .matches(/^[a-zA-Z0-9_-]+$/g, {
```

Comment:
could you pls move this regex to the variable and use in two places as variable?